src/EventGridExtension/EventGridTriggerAttributeBindingProvider.cs

```diff
@@ -41,34 +41,37 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 return Task.FromResult<ITriggerBinding>(null);
             }
 
-            if (!IsSupportedBindingType(parameter.ParameterType))
+
+            var method = typeof(IConverterManager).GetMethod("GetConverter");
+            // we convert data returned from listener or from test API to JObject first
+            var generic = method.MakeGenericMethod(typeof(JObject), parameter.ParameterType, typeof(EventGridTriggerAttribute));
```

Comment:
Do we need the call to MakeGenericMethod? IConverterManager .GetConverter()'s signature is (Type,Type).

Reply:
[`FuncAsyncConverter GetConverter<TAttribute>(Type typeSource, Type typeDest)`](https://github.com/Azure/azure-webjobs-sdk/blob/f70476485521b73f9023ba3b46d8baed5157173c/src/Microsoft.Azure.WebJobs.Host/IConverterManager.cs#L26) only exists in `dev` branch?